components/server/src/redis/publisher.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { inject, injectable } from "inversify";
+import { log } from "@gitpod/gitpod-protocol/lib/util/logging";
+import {
+    PrebuildUpdatesChannel,
+    RedisPrebuildUpdate,
+    RedisWorkspaceInstanceUpdate,
+    WorkspaceInstanceUpdatesChannel,
+} from "@gitpod/gitpod-protocol";
+import { Redis } from "ioredis";
+import { reportUpdatePublished } from "../prometheus-metrics";
+
+@injectable()
+// RedisPublisher is a copy from ws-manager-bridge/src/redis/publisher.go until we find a better
+// way to share the publisher across packages.
+// WEB-621
+export class RedisPublisher {
```

Comment:
Sadly a copy, but this unblocks removal of messagebus and after we can figure out how to share this better